patch.py

```diff
@@ -1,20 +1,121 @@
 #!/bin/python3.6
```

Comment:
not your fault, but this is a bit to specific

Proper way:

```python
#!/usr/bin/env python3
```

Reply:
Damn i noticed it at the beginning and then forgot to fix it. Done now.